app/src/main/kotlin/co/netguru/baby/monitor/client/feature/settings/ConfigurationViewModel.kt

```diff
@@ -74,7 +88,19 @@ class ConfigurationViewModel @Inject constructor(
                     )
                 },
                 onError = { Timber.e(it) }
-            ).addTo(compositeDisposable)
+            )
+    }
+
+    private fun getVoiceAnalysisConfirmationItem(voiceAnalysisOption: VoiceAnalysisOption): ConfirmationItem<VoiceAnalysisOption> {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/c7380d74fc32b4ddceef02c8d9a561008c986f4f/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/settings/ConfigurationViewModel.kt#L94"></span>Line detected that is longer than the defined maximum line length in the code style.</td>
    </tr>
  </tbody>
</table>